Entity/Subscription.php

```diff
@@ -0,0 +1,211 @@
+<?php
+
```

Comment:
Plz remove the auto generated comment and include the standard integrated copyright above every file